src/odemis/driver/test/pmtctrl_test.py

```diff
@@ -247,6 +261,72 @@ def receive_image(self, dataflow, image):
         self.assertEqual(self.pmt.data.active, False)
         self.is_received.set()
 
+class TestTRPMT(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.sem = semcomedi.SEMComedi(**CONFIG_SEM2)
+        cls.control = CLASS_TR_CTRL(**KWARGS_TR_CTRL)
+
+        for child in cls.sem.children.value:
+            if child.name == CONFIG_SED["name"]:
+                cls.sed = child
+            elif child.name == CONFIG_BSD["name"]:
+                cls.bsd = child
+            elif child.name == CONFIG_SCANNER["name"]:
+                cls.scanner = child
+        cls.pmt = CLASS_PMT(name="test", role="detector",
+                                 dependencies={"detector": cls.bsd,
+                                           "pmt-signal": cls.control})
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.pmt.terminate()
+        cls.sem.terminate()
+        cls.control.terminate()
+
+    def tearDown(self):
+        self.logger.removeHandler(self.handler)
+        self.handler.close()
+
+    def setUp(self):
+        # We will need to catch some log messages
+        self.handler = h = TestHandler(Matcher())
+        self.logger = logging.getLogger()
+        self.logger.addHandler(h)
```

Comment:
No need of all this code related to the log handler. That's only useful if you want to check some log messages.